### DISCUSSION TITLE
The windows credentials are known to not work

### DISCUSSION THREAD

**Discussion by eliassal** [dataplat/dbatools#7070] [General]
I have a very basic script to retreive status of sql services (Get-DbaService which works fine), but then when I try to start a service when it is stopped I get

> WARNING: [15:19:28][Get-DbaCmObject] Bad credentials. The windows credentials are known to not work. Exception calling "GetCredential" with "1" ar
> gument(s): "Windows authentication was used, but is known to not work!"
> WARNING: [15:19:28][Update-ServiceStatus] Failed to retrieve service name SQLSERVERAGENT from the CIM object collection - the service will not be 
> processed
> WARNING: [15:19:28][Update-ServiceStatus] (SQLSERVERAGENT on VW19DB1) The attempt to start the service returned the following error: Unknown error
> .

Here is the code

```
$service = Get-DbaService -Credential $winCred -ComputerName $Name -Type Agent,Engine
# Loop through services
ForEach ($dbService in $Service) 
    {
        if ($dbService.State -eq "Stopped")
        {
            $dbService.Start()
            Write-Host "Started service {$dbService.ServiceName.}"
        }
    }
```

I also tried
`(Get-DbaService -Credential $winCred -ComputerName $Name -Type Agent,Engine) | Where State -eq 'Stopped' | Start-DbaService
`
it outputs the same exact result

**Comment by wsmelton** [comment]
This error comes from the section of code below where we are validating the credential can connect to the remote machine using CIM

https://github.com/sqlcollaborative/dbatools/blob/564ad1bac0ccf7b44e31a19874f6b733f112263b/functions/Get-DbaCmObject.ps1#L111-L122

**Comment by eliassal** [comment]
Thanks, so what is the solution? How can I make it work? Do you want to say that we should not use it?
